src/Microsoft.IdentityModel.Tokens/SupportedAlgorithms.cs

```diff
@@ -335,11 +335,6 @@ internal static bool IsSupportedRsaAlgorithm(string algorithm, SecurityKey key)
 
         private static bool IsSupportedRsaPss(SecurityKey key)
         {
-#if NET45
-            // RSA-PSS is not available on .NET 4.5
-            LogHelper.LogInformation(LogMessages.IDX10692);
-            return false;
-#elif NET461 || NET462 || NET472 || NETSTANDARD2_0 || NET6_0_OR_GREATER
```

Comment:
Do we want to have a `#if` here?
to match with the `#else`, line 358

Reply:
This elif seems to cover all of the targets and doesn't feel necessary, but if I'm mistaken then I'm happy to replace it.